lib/appSession.js

```diff
@@ -0,0 +1,88 @@
+const { strict: assert } = require('assert');
```

Comment:
Encryption added and reviewed by security here:

https://github.com/joshcanhelp/express-openid-connect/pull/1